ocpi-tariffs/src/types/money.rs

```diff
@@ -73,15 +73,15 @@ impl Add for Money {
     type Output = Self;
 
     fn add(self, rhs: Self) -> Self::Output {
-        Self((self.0 + rhs.0).into())
+        Self(self.0 + rhs.0)
```

Comment:
You could also have saturating ops here

Reply:
This uses the the `Add` impl of OCPI specific `Number` that already uses saturating. As does almost every other numeric type in the crate.